scripts/extensions/sams/src/ui/grid/GridItem.tsx

```diff
@@ -18,7 +19,7 @@ export class GridItem extends React.PureComponent<IProps> {
         );
 
         return (
-            <div className={classes} onClick={this.props.onClick}>
+            <div className={classes} onClick={this.props.onClick} onDoubleClick={this.props.onDoubleClick}>
```

Comment:
When handling double clicks, it makes for a smoother user experience if we place a `window.setTimeout` over the single click prop (otherwise when double clicking, the single click event will still be fired), and clear this timeout on double click.

Example usage (see`select` and `dbClick` functions):
https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/apps/search/components/Item.tsx#L166